### PR TITLE
Fix CI [changelog skip]

### PR DESCRIPTION
**Pull request by MSP-Greg**
### Description

Fix CI

1. Add racc to Gemfile for JRuby RuboCop
2. test_busy_worker - skip teardown
3. test_integration_single - patch from PR 2560

Please describe your pull request. Thank you for contributing! You're the best.

### Your checklist for this pull request

- [x] I have reviewed the [guidelines for contributing](../blob/master/CONTRIBUTING.md) to this repository.
- [x] I have added an entry to [History.md](../blob/master/History.md) if this PR fixes a bug or adds a feature. If it doesn't need an entry to HISTORY.md, I have added `[changelog skip]` or `[ci skip]` to the pull request title.
- [ ] I have added (or updated) appropriate tests if this PR fixes a bug or adds a feature.
- [x] My pull request is 100 lines added/removed or less so that it can be easily reviewed.
- [ ] If this PR doesn't need tests (docs change), I added `[ci skip]` to the title of the PR.
- [ ] If this closes any issues, I have added "Closes `#issue`" to the PR description or my commit messages.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed, including Rubocop.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -15,6 +15,8 @@ gem "minitest-stub-const"
 gem "sd_notify"
 
 gem "jruby-openssl", :platform => "jruby"
+# needed for rubocop
+gem "racc", :platform => "jruby"
```
Comment (dentarg) [inline Gemfile:19]: Is this a bug in rubocop that we need to add it? (Or bundled or JRuby)

Comment (MSP-Greg) [reply Gemfile:19]: Probably JRuby, but it's recent, and will probably be fixed in the future?  Log from Actions:
```
Run bundle exec rake rubocop
rake aborted!
LoadError: no such file to load -- racc/info
/home/runner/.rubies/jruby-9.2.15.0/bin/bundle:23:in `<main>'
```

**Comment by olleolleolle** [comment]
Now the JRuby release should include racc on its own again, can this work without the intervention?

**Comment by MSP-Greg** [comment]
@olleolleolle

Thanks.  I just checked in my fork.

We bypass RuboCop on JRuby head, probably due to time.  I just downloaded the latest Windows release, and the file is included.

But, jruby-9.2.15.0 doesn't have racc installed or the file is still missing.  See:

https://github.com/MSP-Greg/puma/actions/runs/621117884

Does that seem correct?

**Comment by olleolleolle** [comment]
https://www.jruby.org/2021/03/03/jruby-9-2-16-0.html I was thinking of this release, which _seeeeems_ to mention racc in its notes?

**Comment by MSP-Greg** [comment]
Everyone can keep an eye on it, but 9.2.16.0 isn't available yet from [`ruby/setup-ruby`](https://github.com/ruby/setup-ruby/blob/master/ruby-builder-versions.js).

See https://github.com/ruby/setup-ruby/issues/156

**Comment by MSP-Greg** [comment]
@olleolleolle

Sorry, I should have 'followed' the bug.

I was working on other things and also irritated because 'Ruby packagers' should **never remove** default gems.  There's enough issues with bundled gems (and default gems being converted to bundled gems)...

**Comment by MSP-Greg** [comment]
Fixed in ca434803bc

Thanks to @eregon for the quick updates to both ruby/ruby-builder and ruby/setup-ruby.

**Pull request merged**